src/app/project/ontology/property-info/property-info.component.ts

```diff
@@ -0,0 +1,115 @@
+import { AfterContentInit, Component, Input, OnInit } from '@angular/core';
+import { MatIconRegistry } from '@angular/material/icon';
+import { DomSanitizer } from '@angular/platform-browser';
+import {
+    ApiResponseData,
+    Constants,
+    IHasProperty,
+    ListsResponse,
+    ReadOntology,
+    ResourcePropertyDefinitionWithAllLanguages
+} from '@dasch-swiss/dsp-js';
+import { CacheService } from 'src/app/main/cache/cache.service';
+import { Category, DefaultProperties, PropertyType } from '../default-data/default-properties';
+import { Property } from '../resource-class-form/resource-class-form.service';
+
+@Component({
+    selector: 'app-property-info',
+    templateUrl: './property-info.component.html',
+    styleUrls: ['./property-info.component.scss']
+})
+export class PropertyInfoComponent implements OnInit, AfterContentInit {
+
+    @Input() propDef: ResourcePropertyDefinitionWithAllLanguages;
+
+    @Input() propCard: IHasProperty;
+
+    @Input() projectcode: string;
```

Comment:
projectcode -> projectCode

Reply:
it's the same as in other components. To be consistent, I have to change it everywhere. But I can do it...

Reply:
I've noticed it elsewhere as well. Maybe leave it for now and we can open a small PR to rename them everywhere all at once.